### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by ntrncic**
fixed add new partneer
when adding trips anyone can submit. but only approvers can approve
changed trips api

### CODE REVIEW

File: EquiTrack/trips/admin.py [hunk 1]
```diff
@@ -252,7 +253,7 @@ def get_readonly_fields(self, request, trip=None):
             u'approved_date'
         ]
 
-        if trip and trip.status == Trip.PLANNED and request.user in [trip.owner]:
+        if trip and trip.status == Trip.PLANNED:
```
Comment (jamescw) [inline EquiTrack/trips/admin.py]: How will users submit trips?

Comment (ntrncic) [inline EquiTrack/trips/admin.py]: anyone can submit the users trip now

Comment (jamescw) [inline EquiTrack/trips/admin.py]: we should restrict to owner and supervisor

Comment (jamescw) [inline EquiTrack/trips/admin.py]: and maybe travel focal point, just add all the names in the list

**Pull request merged**